cmake/common.cmake

```diff
@@ -44,17 +44,17 @@ function(enable_coverage name)
   endif()
 endfunction()
 
-set(CURVE_CHOICE "secp384r1" CACHE STRING "One of secp384r1, curve25519, secp256k1_mbedtls, secp256k1_bitcoin")
-if (${CURVE_CHOICE} STREQUAL "secp384r1")
-  add_definitions(-DCURVE_CHOICE_SECP384R1)
-elseif (${CURVE_CHOICE} STREQUAL "curve25519")
-  add_definitions(-DCURVE_CHOICE_CURVE25519)
-elseif (${CURVE_CHOICE} STREQUAL "secp256k1_mbedtls")
-  add_definitions(-DCURVE_CHOICE_SECP256K1_MBEDTLS)
-elseif (${CURVE_CHOICE} STREQUAL "secp256k1_bitcoin")
-  add_definitions(-DCURVE_CHOICE_SECP256K1_BITCOIN)
+set(LEDGER_CURVE_CHOICE "secp384r1" CACHE STRING "One of secp384r1, curve25519, secp256k1_mbedtls, secp256k1_bitcoin")
```

Comment:
Why ledger curve? The ledger is encrypted with symmetric encryption, this is only for identities.

Reply:
I don't know why I ended up with `LEDGER` here, `SERVICE_IDENTITY` is much better.